chemprop/train/loss_functions.py

```diff
@@ -363,3 +351,13 @@ def evidential_loss(pred_values, targets, lam=0, epsilon=1e-8):
     loss = L_NLL + lam * (L_REG - epsilon)
 
     return loss
+
+
+def quantile_loss_batch(pred_values, targets, quantiles):
+    """
+    Batched pinball loss at desired quantiles.
+    """
+
+    error = targets - pred_values
```

Comment:
I think error sign convention is usually the reverse of this, preds - targets?

Reply:
Changed this